app/models/work_version.rb

```diff
@@ -5,7 +5,7 @@ class WorkVersion < ApplicationRecord
   include AggregateAssociations
 
   belongs_to :work
-  has_many :contributors, dependent: :destroy, class_name: 'Contributor'
+  has_many :contributors, inverse_of: :work_version, dependent: :destroy, class_name: 'Contributor'
```

Comment:
This is something that rails can infer without overriding the default:
```ruby
WorkVersion.reflect_on_association(:contributors).inverse_of
=>
#<ActiveRecord::Reflection::BelongsToReflection:0x000000011249a300
 @active_record=
  AbstractContributor(id: integer, work_version_id: integer, first_name: string, last_name: string, created_at: datetime, updated_at: datetime, contributor_type: string, role: string, full_name: string, type: string, weight: integer, orcid: string),
 @class_name="WorkVersion",
 @foreign_key="work_version_id",
 @inverse_name=nil,
 @klass=
  WorkVersion(id: integer, version: integer, title: string, work_type: string, created_edtf: string, abstract: text, citation: string, access: enum, embargo_date: date, license: string, created_at: datetime, updated_at: datetime, state: string, published_edtf: string, subtype: text, work_id: integer, description: string),
 @name=:work_version,
 @options={},
 @plural_name="work_versions",
 @scope=nil>
```

Reply:
Do you know if is there a reason we have it on the `has_many :authors` declaration on the very next line?  Maybe we should remove it there so at least they are consistent.